compiler/dtype.cc

```diff
@@ -175,6 +175,9 @@ int Dtype::SizeOf() const {
             return 4;
         case kFloat64:
             return 8;
+        case kString:
+            // TODO(take-cheeze): Make size accurate
```

Comment:
FYI, I guess it was more appropriate to have `Type::Kind::kString` instead of here?

Reply:
Seems better. Though ONNX doesn't have string kind in value so may need some trick